src/upgrade.yml

```diff
@@ -3,5 +3,32 @@
   name: Upgrade base image
   become: yes
   become_method: sudo
-  roles:
-    - upgrade
+  tasks:
+    # TODO: Remove this block once the upstream kernel bug is fixed.
+    # See https://github.com/cisagov/kali-packer/issues/131 for more
+    # details.
+    #
+    # There is currently a bug in the 6.x kernel that prevents ext4
```

Comment:
Do we need to worry about the new kernel version becoming the version pre-installed in the base AMI?

Reply:
Yes, that would hurt us.  "Fortunately" [the Kali base AMI names do not allow for the automatic "grab the latest" behavior that we leverage with the Debian base AMIs](https://github.com/cisagov/kali-packer/blob/ecc7a3e401c6a1a8bff818cd275b75766b0e4fa2/src/packer.pkr.hcl#L61), so I would have to manually update the base AMI before it would be a problem.  If I did that then there would be a full round of testing, so we should be OK.

Reply:
BTW, Kali is based on Bookworm not Sid.  Bookworm is the current Debian _Testing_, while Sid is the name for Debian _Unstable_.  (Debian Testing is what will become the next release, so it is assigned a name; Debian Unstable is an unnamed purgatory.)

If Kali were based on Sid then we'd be lucky if it _ever_ worked!